modules/pilotxBidAdapter.js

```diff
@@ -0,0 +1,147 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+const BIDDER_CODE = 'pilotx';
+const ENDPOINT_URL = '//adn.pilotx.tv/hb'
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: ['banner', 'video'],
+  aliases: ['pilotx'], // short code
+  /**
+         * Determines whether or not the given bid request is valid.
+         *
+         * @param {BidRequest} bid The bid params to validate.
+         * @return boolean True if this is a valid bid, and false otherwise.
+         */
+  isBidRequestValid: function (bid) {
+    let sizesCheck = !!bid.sizes
+    let paramSizesCheck = !!bid.params.sizes
+    var sizeConfirmed = false
+    if (sizesCheck) {
+      if (bid.sizes.length < 1) {
+        return false
+      } else {
+        sizeConfirmed = true
+      }
+    }
+    if (paramSizesCheck) {
+      if (bid.params.sizes.length < 1 && !sizeConfirmed) {
+        return false
+      } else {
+        sizeConfirmed = true
+      }
+    }
+    if (!sizeConfirmed) {
+      return false
+    }
+    return !!(bid.params.placementId || (bid.params.member && bid.params.invCode));

```

Comment:
Hi @AnthonyBoozan, sorry for the delay, I don't see `bid.params.member` and `bid.params.invCode` mentioned in the docs PR, also I don't see it in the `.md` file bellow?